build_local.sh

```diff
@@ -113,85 +101,41 @@ if [[ ${CROSS_OVER_VERSION} == 20.* ]]; then
     pushd sources/wine
     patch -p1 < ${GITHUB_WORKSPACE}/msvcrt-sincos.patch
     popd
-
-    echo Patch DXVK
-    patch sources/dxvk/src/util/rc/util_rc_ptr.h < dxvk_util_rc_ptr.patch
 fi
 
-############ Build LLVM / Clang ##############
```

Comment:
Why don't you build LLVM and Clang any more?

Reply:
That was replaced by using cx-llvm

Reply:
Oh, I did not spot the cx-llvm. This is pretty nice.